src/parsers/manifestjson.js

```diff
@@ -246,31 +269,14 @@ export default class ManifestJSONParser extends JSONParser {
         this.collector.addWarning(messages.WRONG_ICON_EXTENSION);
       } else {
         promises.push(
-          getImageMetadata(this.io, _path)
-            .then((info) => {
-              if (info.width !== info.height) {
-                this.collector.addError(messages.iconIsNotSquare(_path));
-                this.isValid = false;
-              } else if (info.mime !== 'image/svg+xml' &&
-                         parseInt(info.width, 10) !== parseInt(size, 10)) {
-                this.collector.addWarning(messages.iconSizeInvalid({
-                  path: _path,
-                  expected: parseInt(size, 10),
-                  actual: parseInt(info.width, 10),
-                }));
-              }
-            })
-            .catch(() => {
-              this.collector.addWarning(messages.corruptIconFile({
-                path: _path,
-              }));
-            })
+          this.validateIcon(_path, size)
         );
       }
     });
     return Promise.all(promises);
   }
 
+
```

Comment:
Nit, remove this empty line

Reply:
Weird that eslint isn't warning here, hmm

Reply:
We have an explicit eslint rule in web-ext to allow it:

https://github.com/mozilla/web-ext/blob/7832938473bea398768c35a093fdc11395923369/.eslintrc#L95

it doesn't seem to be part of the addons-linter eslint config, and it doesn't seem to be part of the rules coming from the eslint-config-amo preset, but it is part of the config if I run "./node_modules/.bin/eslint --print-config .", and the following is its current value:

```
    "no-multiple-empty-lines": [
      "error",
      {
        "max": 2,
        "maxEOF": 1
      }
    ],
```